docs/openstack-openedx/supported-deployment.md

```diff
@@ -0,0 +1,99 @@
+# OpenStack Production Deployment (Proposal)
+
+edX does not officially provide any supported method for deploying a production-grade instance of Open edX. However, the following *proposed* method for setting up a production-grade instance on OpenStack is available to the community, and support for this method will be provided by OpenCraft, if there is sufficient community interest in this approach. For context, this method of deployment is used in production for over 100 production instances of Open edX on [OpenCraft's hosting platform](http://opencraft.com/hosting/).
+ 
+
+## Features of this Deployment Method
+With this method, you'll get an instance of Open edX that's ready to handle production traffic and which follows well-established best practices for deployments.
+ 
+Details:
+* Uses the latest stable Open edX release (Ficus)
+* Can be scaled to a multi-VM setup to handle higher traffic levels
+* Uses ansible for automated, repeatable provisioning of VMs
+* Separates state (user data, course data, etc.) from the app servers (LMS, Studio, etc.)
+* Allows most updates and upgrades to be completed without any downtime
+* Fully open-source stack
+* Does not currently support ecommerce, course discovery, or Insights
+ 
+Requirements
+* In order to use this deployment method, you will need:
+* An OpenStack cloud provider (Juno release or newer) including Compute (Nova) and Block * Storage (Swift). A provider that supports Neutron networking and offers load balancing as a * service (LBaaS) is highly recommended.
+  - Public cloud providers are listed in the [OpenStack Marketplace](https://www.openstack.org/marketplace/public-clouds/).
+  - At a minimum, you will need a VM with 4 GB of RAM
+* A MySQL server/cluster running MySQL version 5.6 or newer
+* A MongoDB server/cluster running MongoDB version 2.6, 3.0, or 3.2
+* A VM running RabbitMQ
+* A VM running ElasticSearch version 0.90.13 (not any newer version) - may be installed on the * same VM as RabbitMQ
+* An SSL certificate for each of the domains (typically there are three: www.example.com, studio.example.com, preview.example.com). We recommend using Let's Encrypt for free SSL certificates.
+ 
+As these are standard open source services, **the installation, configuration, and operation of these database servers is left to your operations team**, and would not be supported by OpenCraft nor edX. However, ansible playbooks which may be used to set up most of these services are provided at https://github.com/edx/configuration/tree/master/playbooks (edX) and https://github.com/open-craft/deployment-deploy-databases (OpenCraft).
+ 
+Additional notes:
+* All of the databases and services like RabbitMQ and ElasticSearch should be firewalled off from the public internet.
+* Be sure to implement some sort of regular backup for the data in MySQL, MongoDB, and Swift.
+
+
+## How to deploy for the first time
```

Comment:
No. As you can see, it's a ton of work to go through this, so I wanted to gauge interest first. Testing this thoroughly would probably take 1-2 days?

I did test a very similar process on a recent client setup, but I haven't tested these specific instructions, and there is no doubt some minor issues with it that will need to be fixed.